auslib/blobs/apprelease.py

```diff
@@ -310,6 +318,7 @@ def __init__(self, **kwargs):
         Blob.__init__(self, **kwargs)
         if 'schema_version' not in self.keys():
             self['schema_version'] = 1
+        self['has_wnp'] = self.has_wnp(field='openURL')
```

Comment:
This is _not_ (and shouldn't be) part of the blob data, so it's actually better off as a property or plain function that looks at openURL when it's called. By adding it as part of the blob, there's potential for it to get out of sync with the state of the opneURL field.

Reply:
Which part do you think is expensive? Figuring out whether or not a WNP exists on a blob should be cheap. For example, for ReleaseBlobV9, you just need to walk through self.updateLine and look for any "openURL" keys.

The expensive part is retrieving the actual blobs from the database (which we don't currently do when building the Rules page). For now, I think it's OK to just go ahead and do it, and compare the before and after performance. We have an in-memory cache for Releases, and that may end up making this pretty fast. If it causes a significant slow down, we can consider other options.

If you need any other clarification, let's chat on IRC.